src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --header-secondary-color: #00acdc;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: block;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__ul {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__link:link {
```

Comment:
Have noticed - what is `.nav__link:link`? Perhaps you forgot to delete `:link `

Reply:
I've used :link pseudo class for more consistent styles format, I've learnt it in [this](https://mate.academy/learn/css-basics/links-and-urls#/video/852) lecture, but anyway I've already deleted it, as you told me